.gitlab-ci.yml

```diff
@@ -430,3 +430,22 @@ bump-chrome-version-scheduled:
     - eval $(ssh-agent -s)
   script:
     - node scripts/bump-chrome-driver-version.js
+
+bump-chrome-version-scheduled-success:
```

Comment:
Next hackaton: use a node library to send slack notifications :D

Reply:
Yes totally! I though the same while writing it :)

Reply:
don't hesitate to create a task to keep track of the idea 😉